proposals/2022-04-upgrade-v2-0-3-min-commission-rate/README.md

```diff
@@ -0,0 +1,25 @@
+# Upgrade the chain to v2.0.3 for introducing a min-commission-rate parameter and setting it to 3%
+
+
+### Proposal
+
+This proposal is for upgrading the Panacea mainnet `panacea-3` from v2.0.2 to v2.0.3 for introducing a min-commission-rate parameter and setting it to 3%.
+
+A signaling proposal for this had been already approved: https://www.mintscan.io/medibloc/proposals/2.
```

Comment:
```suggestion
The signaling proposal for this change has already been approved: https://www.mintscan.io/medibloc/proposals/2.
```

Reply:
uri of approved signaling proposal have to be changed.
```suggestion
A signaling proposal for this had been already approved: https://www.mintscan.io/medibloc/proposals/7.
```